csv-to-genesis/src/main/kotlin/jp/co/soramitsu/Converter.kt

```diff
@@ -0,0 +1,153 @@
+package jp.co.soramitsu
+
+import jp.co.soramitsu.iroha2.Genesis
+import jp.co.soramitsu.iroha2.JSON_SERDE
+import jp.co.soramitsu.iroha2.asName
+import jp.co.soramitsu.iroha2.asValue
+import jp.co.soramitsu.iroha2.cast
+import jp.co.soramitsu.iroha2.client.Iroha2Client
+import jp.co.soramitsu.iroha2.generated.core.genesis.GenesisTransaction
+import jp.co.soramitsu.iroha2.generated.core.genesis.RawGenesisBlock
+import jp.co.soramitsu.iroha2.generated.datamodel.Name
+import jp.co.soramitsu.iroha2.generated.datamodel.account.AccountId
+import jp.co.soramitsu.iroha2.generated.datamodel.asset.AssetDefinitionId
+import jp.co.soramitsu.iroha2.generated.datamodel.asset.AssetId
+import jp.co.soramitsu.iroha2.generated.datamodel.asset.AssetValueType
+import jp.co.soramitsu.iroha2.generated.datamodel.isi.Instruction
+import jp.co.soramitsu.iroha2.transaction.Instructions
+import jp.co.soramitsu.iroha2.transaction.TransactionBuilder
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVParser
+import org.apache.commons.csv.CSVRecord
+import java.io.File
+import java.math.BigInteger
+import java.net.URL
+import java.security.KeyPair
+import java.text.SimpleDateFormat
+import java.time.Duration
+import java.util.*
+
+class Converter {
+    companion object {
+        private val ID = "id".asName() to 0
+        private val FRAUD_TYPE = "ft".asName() to 12
+        private val ORIGINATION = "org".asName() to 4
+        private val DESTINATION = "dst".asName() to 5
+        private val STATUS = "sts".asName() to 14
+        private val TIMESTAMP = "ts".asName() to 1
+        private val EXPIRY_DATE = "ed".asName() to TIMESTAMP.second
+
+        val dateFormatter = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+    }
+
+    suspend fun sendToIroha(csv: File, peerUrl: URL, admin: AccountId, keyPair: KeyPair) {
+        val client = Iroha2Client(peerUrl, true)
+        csv.bufferedReader().use { reader ->
+            val parser = CSVParser(
+                reader, CSVFormat.DEFAULT
+                    .withFirstRecordAsHeader()
+                    .withSkipHeaderRecord()
+            )
+            parser.forEach {
```

Comment:
so we construct a separate transaction for each raw and them individually? could we merge them to one transaction first and send atomically?